Added PR template, Security Policy and CODEOWNERS (#26)
HaveIBeenPwned/PwnedPasswordsAzureFunction: .github/PULL_REQUEST_TEMPLATE.md

## Description

<!--
Please include a summary of the change(s) and/or  which issue(s) are fixed. 
This includes of any changes to current behaviour, new services required, and any breaking changes
If this closes an issue, please include "closes #XXXX" in your comment to auto-close the issue
-->

## Checklist:
<!-- Please replace every instance of `[ ]` with `[X]` to indicate you have completed the criteria -->

> - [ ] I confirm that my submission adheres to the [Code of Conduct](https://github.com/HaveIBeenPwned/PwnedPasswordsAzureFunction/blob/main/CODE_OF_CONDUCT.md).
> - [ ] I have performed a self-review of all of my code
> - [ ] I have ensured all of my code has `<summary>` tags for all methods and public variables, and has comments to understand program execution flow where it is hard to understand, as well as updating any `<summary>` tags in functions where functionality has changed.
> - [ ] I have tested that the code works with sample data running the code either locally or in my own deployed Azure Function.
> - [ ] I have verified that any tests that I have supplied with this Pull Request work and I have fixed any code which caused pre-existing tests to fail.
> - [ ] I have updated the [README](https://github.com/HaveIBeenPwned/PwnedPasswordsAzureFunction/blob/main/README.md) with any changes in dependencies/configuration values.
> - [ ] I have linted my code to ensure that it is formatted correctly.

Reviewer: Experience has learned me that your PR templates should be short or nobody will actually bother, so I'd propose the following:

```suggestion
- [ ] I confirm that my submission adheres to the [Code of Conduct](https://github.com/HaveIBeenPwned/PwnedPasswordsAzureFunction/blob/main/CODE_OF_CONDUCT.md).
- [ ] I have performed a self-review of all of my code
- [ ] I have tested that the code works with sample data and verified all tests are passing
- [ ] I have updated the [README](https://github.com/HaveIBeenPwned/PwnedPasswordsAzureFunction/blob/main/README.md) with any changes in dependencies/configuration values.
- [ ] I have linted my code to ensure that it is formatted correctly.
```
